.github/workflows/build.yml

```diff
@@ -39,19 +58,34 @@ jobs:
             libssl-dev \
             gcc-multilib \
             libelf-dev \
-      - uses: actions/checkout@v4
-      - uses: actions/checkout@v4
+
+      - uses: taiki-e/setup-cross-toolchain-action@v1
```

Comment:
What does this action do? Does it distill down into just adding toolchains?

Reply:
Looks like I was able to build without it, so I'll remove it.

Reply:
Done